staging/src/k8s.io/apimachinery/pkg/util/runtime/runtime.go

```diff
@@ -94,12 +94,12 @@ var ErrorHandlers = []func(error){
 	}).OnError,
 }
 
-// HandlerError is a method to invoke when a non-user facing piece of code cannot
-// return an error and needs to indicate it has been ignored. Invoking this method
-// is preferable to logging the error - the default behavior is to log but the
-// errors may be sent to a remote server for analysis.
+// HandleError handles an error by a list of pre-defined error handlers, including
+// logging and sending to a remote server for analysis, and thus calling this
+// function is preferable to logging the error. It is for the case when the error
+// can't be returned or should be ignored.
```

Comment:
I think `and has been ignored` matches the previous comment more than `should be ignored`

Reply:
done. PTAL